awx/ui_next/src/screens/Application/ApplicationDetails/ApplicationDetails.jsx

```diff
@@ -21,11 +21,11 @@ function ApplicationDetails({
   const history = useHistory();
   const {
     isLoading: deleteLoading,
-    deletionError,
-    deleteItems: handleDeleteApplications,
+    error: deletionError,
+    request: deleteApplications,
     clearDeletionError,
```

Comment:
useRequest doesn't return a `clearDeletionError`. You'll have to pass `deletionError` into a `useDismissableError` below this to get the clearDeletionError function